ui/admin/tests/acceptance/host-catalogs/list-test.js

```diff
@@ -215,6 +224,21 @@ module('Acceptance | host-catalogs | list', function (hooks) {
     assert.dom(`[href="${urls.azureHostCatalog}"]`).exists();
   });
 
+  test('user can search for gcp host catalog', async function (assert) {
+    await visit(urls.projectScope);
+
+    await click(`[href="${urls.hostCatalogs}"]`);
+    assert.dom(`[href="${urls.staticHostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.awsHostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.azureHostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.gcpHostCatalog}"]`).exists();
+
+    await fillIn(SEARCH_INPUT_SELECTOR, TYPE_HOST_CATALOG_PLUGIN_GCP);
+    await waitFor(`[href="${urls.staticHostCatalog}"]`, { count: 0 });
+
+    assert.dom(`[href="${urls.gcpHostCatalog}"]`).exists();
```

Comment:
Should we change the `.exists()` to `.isVisible()`?

Reply:
good catch, updated!